app/styles/components/_nypr-player-overrides.scss

```diff
@@ -48,3 +48,57 @@ $nypr-player-small-only                      : "(max-width: #{$nypr-player-mediu
 .nypr-player-wrapper.liquid-container {
   transform: none;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 25px;
+  border-bottom: dotted 1px #0064b4;
+  svg {
+    min-width: 220px;    
+  }
+  .live-text-box {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: white;
+    height: 14px;
+    min-width: 99px;
+    border-radius: 5px;
+    color: #0064b4;
+    margin: 0 99px;
```

Comment:
generally speaking with CSS, you don't want an element to lay itself out. imagine if you used `.live-text-box` in a different part of the interface. this margin rule would probably give you an undesired result.

the better practice would be to put `.live-text-box` inside another element that handles the layout, something like `.centered-block` or something, where `.centered-block` sets the margins and positioning.

also: why `0 99px`? are you trying to center something?